src/celeritas/phys/PhysicsStepUtils.hh

```diff
@@ -105,6 +105,18 @@ calc_physics_step_limit(const MaterialTrackView& material,
             }
         }
     }
+    else
+    {
+#if !CELER_DEVICE_COMPILE
+        CELER_VALIDATE(total_macro_xs > 0,
+                       << "non-positive cross section (" << total_macro_xs
+                       << ") for stopped particle of type "
+                       << particle.particle_id().get() << " in material "
+                       << material.material_id().get() << " with "
+                       << physics.num_particle_processes()
+                       << " possible processes");
+#endif
```

Comment:
Yes, due to a bug in the "along step without fluctuation" kernel. It should be safe to leave in for debugging since I think we've hit some strange errors with stopped particles: but since it is not keeping to the spirit of `CELER_VALIDATE`, should I delete it? Or just change to an assertion?

Reply:
Gotcha, fixed by those changes to the eloss applier I guess? Sure, maybe make it an assertion?

Reply:
Exactly and yep!